Beatmap descriptions (#3772)
ppy/osu-wiki: wiki/Beatmaps/Beatmap_description/en.md

---
stub: true
---

> ## Beatmap description

Reviewer: ```suggestion
# Beatmap descriptions
```

**Beatmap descriptions** are small areas under a [beatmap](/wiki/Beatmaps)'s page that give players information about said beatmap using [BBCode](/wiki/BBCode). Most [Ranked](/wiki/Beatmaps#ranked) beatmaps use at least one image, a number (depending on how many beatmaps the creator has Ranked), and a difficulty section.

> ## Good beatmap descriptions

Reviewer: ```suggestion
## Uses and information
```

> *Notice: In order to get a beatmap ranked, it is recommended you have a good beatmap description, but this is not required.*

Reviewer: ```suggestion
*Notice: In order to get a beatmap ranked, it is recommended that creators have a good beatmap description, but this is not required.*
```

> A good beatmap description would include:

Reviewer: ```suggestion
Popular beatmap descriptions often include the following:
```

- A list of all the guest difficulties and their creators
- An "18+ lyrics" warning (if neccessary)
- Information that would help a player decide whether they want to download the beatmap or not
- An edits list or changelog

### Images

> Many beatmap descriptions contain at least one image. To learn know how to add a image using BBCode, check out the [BBCode article](/wiki/BBCode#image). Also, make sure you have the link of the image you want to put in the beatmap description.

Reviewer: ```suggestion
Many beatmap descriptions contain at least one image (often an animated [GIF](https://en.wikipedia.org/wiki/GIF "Wikipedia") of some sort), that are used to express and illustrate either the creator, song, and/or beatmap in a flashly way. They can also take places of a title and/or header to add flair and quality. To learn how to add an image using BBCode, check out the [BBCode article's tutorial on images](/wiki/BBCode#image).
```
mostly just added more information here

> ### Difficulty sections
> 
> A difficulty section is located in a beatmap description, usually this section has a list of all the guest difficulties and their creators, this is mostly used to showcase the difficulties of a beatmap. Difficulty sections usually have images for the icons.
> 
> #### Difficulty icons
> 
> The difficulty icons can be found in the [Difficulties article](/wiki/Difficulties).

Reviewer: ```suggestion
### Difficulty sections

A "difficulty section" is a list of all the guest difficulties and their creators located in a beatmap description. This is mostly used to showcase the difficulties of a beatmap. Difficulty sections usually have images for the icons.

*Note: The difficulty icons can be found in the [Difficulties article](/wiki/Difficulties).*
```
